test/test_jit.py

```diff
@@ -19572,8 +19572,37 @@ def wrong4(x):
                 # type: (OneTwoWrong) -> int
                 return as_interface(x)
 
-    # TODO test: interface-interface class-interface inheritance errors,
-    # NamedTuple inheritance errors
+        # Test interface/class python assignment
+        with torch.jit._disable_emit_hooks():
+            class TestPyAssign(nn.Module):
+                proxy_mod: OneTwo
+
+                def __init__(self):
+                    super(TestPyAssign, self).__init__()
+                    self.proxy_mod = Foo()
+
+                def forward(self, x):
+                    return self.proxy_mod.two(x)
+
+            input = torch.rand(3, 4)
+            scripted_pyassign_mod = torch.jit.script(TestPyAssign())
+            self.assertEqual(scripted_pyassign_mod(input), 2 * input)
+
+            class TestPyAssignError(nn.Module):
+                proxy_mod: OneTwoThree
```

Comment:
python 3 feature, needs to be in test_jit_py3

Reply:
seems I need to copy all the relevant classes into test_jit_py3, I will probably just use `__annotations__` in py2 to get this work in this file.